Forensics-OSINT/_posts/2020-07-14-Alien-Transmission-1.md

```diff
@@ -1,13 +1,18 @@
 ---
 layout: post
-title: "Alien-Transmission-1"
-author: "Anonymous"
+title: "Alien Transmission 1"
+author: "INXS_JOY"
 tags: ['Forensics-OSINT']
 ---
 
-Question Statement
+I was listening to my radio the other day and received this strange message... I think it came from an alien?
 
```

Comment:
Can you attach the files given in the question? For example, in this question, you can add `squeakymusic.wav`